src/App.js

```diff
@@ -25,17 +32,56 @@ class App extends Component {
     })
   }
 
+  changeLanguageSiteUkr() {
+    var btnU = document.getElementById('ukr');
+    var btnK = document.getElementById('kor');
+    this.setState({
+      language: ukr,
+    })
+    btnK.disabled = false;
+    btnK.classList.remove('active-language')
+    btnU.disabled = true;
+    btnU.classList.add('active-language');
+  }
+
+  changeLanguageSiteKor() {
```

Comment:
@Babiy7 some duplicated code with changeLanguageSiteUkr function. Please extract duplications into another function and reuse code.